v23.1/ui-key-visualizer.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Key Visualizer
+summary: The Key Visualizer page provides insight into potential problems and hot ranges in your deployment.
+toc: true
+docs_area: reference.db_console
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include {{ page.version.version }}/ui/admin-access.md %}
+
+{% include_cached new-in.html version="v23.1" %} The **Key Visualizer** page of the DB Console provides access to the key visualizer tool, which enables the visualization of current and historical [key-value (KV)](architecture/distribution-layer.html#table-data-kv-structure) traffic serviced by your cluster.
+
+The key visualizer is a useful troubleshooting tool when experiencing performance problems with your cluster, surfacing historical and current KV [hot spots](performance-best-practices-overview.html#hot-spots) in your keyspace, drawing attention to [range splits](architecture/distribution-layer.html#range-splits), and highlighting potentially-unnecessary [full-table scans](make-queries-fast.html) that might benefit from the creation of a targeted index, among others.
+
+The key visualizer is disabled by default. Once [enabled](#enable-the-key-visualizer), the key visualizer continuously collects keyspace usage data across your cluster in the background at a [configurable sampling rate](#key-visualizer-customization). Data shown in the key visualizer is retained for a maximum period of seven days.
+
+To access the key visualizer, [navigate to the DB Console](ui-overview.html#db-console-access) and click **Advanced Debug** in the left-hand navigation, then click **Key Visualizer** under the **Even More Advanced Debugging** heading.
+
+## Required privileges
+
+To view the **Advanced Debug** page, and work with the key visualizer, the user must be a member of the `admin` role or must have the `VIEWDEBUG` [system privilege](security-reference/authorization.html#supported-privileges) defined.
+
+## Enable the key visualizer
+
+To use the key visualizer, the `keyvisualizer.enabled` [cluster setting](cluster-settings.html) must be set to `true`, using the [`SET CLUSTER SETTING`](set-cluster-setting.html) statement:
```

Comment:
```suggestion
To use the key visualizer, the [cluster setting](cluster-settings.html) `keyvisualizer.enabled` must be set to `true`, using the [`SET CLUSTER SETTING`](set-cluster-setting.html) statement:
```

Reply:
We do in fact tend to present cluster settings in the way I've presented this one here:

- ```the `.*` \[cluster setting\]``` :: 794 results
- ```the \[cluster setting\](.*) `.*```` : 71 results.

Neither is wrong, neither is perfect, but one is more common.

Reply:
Oh, thanks for checking! I'll open a style issue - I prefer the latter so that people don't have to get through a setting name to know what we're talking about, in some cases.

Reply:
Remember, in the future it **could** look like this!

- ```the [`keyvisualizer.enabled`](cluster-settings.html#setting-keyvisualizer_enabled) cluster setting.```